content/en/blog/_posts/2023-06-29-container-image-signature-verification/index.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: blog
+title: "Verifying container image signatures within CRI runtimes"
```

Comment:
```suggestion
title: "Verifying Container Image Signatures Within CRI Runtimes"
```
non-blocking nit: blog titles should be title case -- although the past few published blogs don't adhere to this, title case for blog titles is the general styling format